### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by RashRAJ**
Added EKS Module and IAM

### CODE REVIEW

**Comment by m3rryqold** [comment]
@RashRAJ,I noticed there are 2 module folders differentiated by naming conventions. Would you consolidate this later? Also, can you add a update the readme with some basic usage instructions? Eg. Steps for trying out a module.

File: AWS Module/EC2 Instance/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+provider "aws" {
+    access_key = "${var.access_key}"
+    secret_key = "${var.secret_key}"
+    region = "${var.region}"
+}
```
[the remark this answers is not in the document]
Comment (RashRAJ) [reply AWS Module/EC2 Instance/main.tf:5]: oh that's the old one just realized when I changed the folder name it created a duplicate of the previous one and my changes wasn't reflected, ill correct that now

**Comment by RashRAJ** [comment]
> @RashRAJ,I noticed there are 2 module folders differentiated by naming conventions. Would you consolidate this later? Also, can you update the `readme` with some basic usage instructions? Eg. Steps for trying out a module.

yeah I just noticed the module repetition i'll fix that now, i will work on the readme now

File: AWS Module/EC2 Instance/main.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (RashRAJ) [reply AWS Module/EC2 Instance/main.tf]: fixed

**Review by m3rryqold — COMMENTED** [review]
Great job on the readme. A user can kickoff with this. We can work on optimizations later.
cc: @Timtech4u

**Comment by m3rryqold** [comment]
@RashRAJ, are the modules accessible externally? Eg.if Ib wants to reference it from another directory.

**Pull request merged**